packages/compass-query-bar/src/components/generative-ai/ai-text-input.tsx

```diff
@@ -224,6 +269,7 @@ const ConnectedAITextInput = connect(
     };
   },
   {
+    cancelAIQuery: cancelAIQuery,
```

Comment:
I think this can be `onCancelAIQuery` to make it named more callback prop like and still be descriptive of when this will be called, no strong preference though if you want to keep it like that

Reply:
Sounds good to me, will rename in the next pr.